src/linter.js

```diff
@@ -64,37 +70,42 @@ export default async function linter(options, compiler) {
     'ESLintWebpackPlugin',
     async (compilation) => {
       if (warnings.length > 0) {
+        // @ts-ignore
```

Comment:
Need avoid it

Reply:
the type for those lists is WebpackError, which appears to be a private class, is there a trick to making tsc happy?

Reply:
we need to added this to webpack types